### ISSUE TITLE
Feature Request : opening Multiple Containers with same URL/Website

### ISSUE THREAD

**Issue by sunilgarg7** [mozilla/multi-account-containers#1956]
People maintaining multiple Identity require an option to open multiple containers with the same URL/website. A kind of "Reopen this tab in all container or selected container" option.
alternatively we can provide more advance user an option to define tabs in Json format or some other format in some specific setting bar.

**Comment by goatllama** [comment]
Yes! This is what I came here to say as well. I thought this was touted as a feature from the get-go but have never been able to get it to work. I have two different outlook accounts I access for junk mail management through the web app and it would be great to not have to sign out/in all the time, especially since outlook makes it such a dang hassle.

**Comment by MichaelKetting** [comment]
I believe this is a duplicate of https://github.com/mozilla/multi-account-containers/issues/1749

**Comment by SpinB** [comment]
Or the possibility of opening a domain with a already assigned container to open on another container, for ex. if you have Domain1 assigned to Container Mail1 using Account1, **right-click and on a Top Site and open Domain 1 in Container Mail2** to use Account2 **without getting automatically redirect to Container Mail1**, do **NOT automatically redirect users to the default container if they choose another one**, but keep using the default one if no container is defined by the user.
